environment/src/main/java/com/sequenceiq/environment/environment/validation/validators/EnvironmentNetworkProviderValidator.java

```diff
@@ -88,14 +88,20 @@ private void evaluateProviderNetworkRelation(boolean networkParamExists, String
         return Optional.ofNullable(o);
     }
 
-    private void validateNetwork(NetworkDto networkDto, String cloudPlatform, ValidationResultBuilder resultBuilder) {
-        if (networkDto != null && Strings.isNullOrEmpty(networkDto.getNetworkCidr())) {
-            EnvironmentNetworkValidator environmentNetworkValidator = environmentNetworkValidatorsByCloudPlatform.get(valueOf(cloudPlatform));
-            if (networkDto.getNetworkCidr() != null && isInvalidNetworkMask(networkDto.getNetworkCidr())) {
-                resultBuilder.error(String.format("The netmask must be /%s.", EXPECTED_NETWORK_MASK));
+    private void validateNetwork(NetworkDto networkDto, String cloudPlatform, ValidationResultBuilder resultBuilder, EnvironmentDto environmentDto) {
+        if (networkDto != null) {
+            if (Strings.isNullOrEmpty(networkDto.getNetworkCidr())) {
+                if (networkDto.getNetworkCidr() != null && isInvalidNetworkMask(networkDto.getNetworkCidr())) {
+                    resultBuilder.error(String.format("The netmask must be /%s.", EXPECTED_NETWORK_MASK));
+                }
+            }
+            if (resultBuilder.build().hasError()) {
+                return;
             }
+            EnvironmentNetworkValidator environmentNetworkValidator = environmentNetworkValidatorsByCloudPlatform.get(valueOf(cloudPlatform));
             if (environmentNetworkValidator != null) {
-                environmentNetworkValidator.validateDuringFlow(networkDto, resultBuilder);
+                LOGGER.debug("Going to validate environment ({}) network in flow...", environmentDto.getName());
```

Comment:
I dont think we need '...' in LOGGER

Reply:
Hehe okey. It’s just a “my kind of” informal stuff, usually I add this triple dot in the logs where something is about to happen which could take a while.
But have it your way :)